applications/nrf5340_audio/boards/nrf5340_audio_dk_nrf5340_cpuapp.overlay

```diff
@@ -41,3 +42,104 @@
 		};
 	};
 };
+
+/ {
+	uac2_headset: usb_audio2 {
+		compatible = "zephyr,uac2";
+		status = "okay";
+		full-speed;
+		audio-function = <AUDIO_FUNCTION_HEADSET>;
+
+		uac_aclk: aclk {
+			compatible = "zephyr,uac2-clock-source";
+			clock-type = "internal-programmable";
+			frequency-control = "read-only";
+			sampling-frequencies = <48000>;
```

Comment:
As we talked about, this is set for feature-parity for now. Please create a task for expanding sampling freq and async transfer.

Reply:
Done